### PR TITLE
Add Magical Gadget category page

### PR DESCRIPTION
**Pull request by svr333**
I followed the layout of the weapons category.
I also asked a question regarding groups of similar items, if that needs to change feel free to point that out,

Ill try and add some of the magical gadgets pages over the next few pages.
Preferably in multiple PRs (looking at you hacktoberfest), but you can leave this one open and for me to add more.

### CODE REVIEW

**Review by LinoxGH — COMMENTED** [review]
Well this is currently fine to merge, although you said yourself that we can keep this open so you can add more pages, well go ahead.

File: pages/Magical-Gadgets.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## Table of contents
+----
+[Ender Backpack](https://github.com/Slimefun/Slimefun4/wiki/Ender-Backpack)  
+[Magic Eye of Ender](https://github.com/Slimefun/Slimefun4/wiki/Magic-Eye-of-Ender)  
+[Elemental Staff](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
+[Elemental Staff - Wind](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
+[Elemental Staff - Water](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
+[Elemental Staff - Fire](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
+[Elemental Staff - Storm](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)
```
Comment (TheBusyBiscuit) [inline pages/Magical-Gadgets.md:11]: The plural of "Staff" is "Staves", not "Staffs".

File: pages/Magical-Gadgets.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+[Elemental Staff - Fire](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
+[Elemental Staff - Storm](https://github.com/Slimefun/Slimefun4/wiki/Elemental-Staffs)  
+[Magical Zombie Pills](https://github.com/Slimefun/Slimefun4/wiki/Magical-Zombie-Pills)  
+[Infused Magnet](https://github.com/Slimefun/Slimefun4/wiki/Infused-Magnets)  
+[Soulbound Backpack](https://github.com/Slimefun/Slimefun4/wiki/Soulbound-Backpack)  
+[Reinforced Spawner](https://github.com/Slimefun/Slimefun4/wiki/Reinforced-Spawner)  
+[Scroll of Dimensional Teleposition](https://github.com/Slimefun/Slimefun4/wiki/Scroll-of-Dimensional-Teleposition)  
+[Tome of Knowledge Sharing](https://github.com/Slimefun/Slimefun4/wiki/Tome-of-Knowledge-Sharing)  
+[Flask of Knowledge](https://github.com/Slimefun/Slimefun4/wiki/Flask-of-Knowledge)  
+[Ancient Altar](https://github.com/Slimefun/Slimefun4/wiki/Ancient-altar)
```
Comment (TheBusyBiscuit) [inline pages/Magical-Gadgets.md:19]: *Ancient-Altar

File: pages/Magical-Gadgets.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+[Elytra](https://github.com/Slimefun/Slimefun4/wiki/Elytra)  
+[Infused Elytra](https://github.com/Slimefun/Slimefun4/wiki/Infused-Elytra)  
+[Soulbound Elytra](https://github.com/Slimefun/Slimefun4/wiki/Soulbound-Elytra)  
+[Totem of Undying](https://github.com/Slimefun/Slimefun4/wiki/Totem-of-Undying)  
+[Rainbow Wool](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Wool)  
+[Rainbow Glass](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Glass)  
+[Rainbow Glass Pane](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Glass-Pane)  
+[Rainbow Clay](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Clay)  
+[Rainbow Concrete](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Concrete)  
+[Rainbow Glazed Terracotta](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Glazed-Terracotta)
```
Comment (TheBusyBiscuit) [inline pages/Magical-Gadgets.md:31]: I think one single "Rainbow-Blocks" page would be fine for this, it's all pretty much the same deal :LUL:

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: pages/Magical-Gadgets.md (hunk 1 above)
Comment (svr333) [reply pages/Magical-Gadgets.md:11]: Ive been living a lie

**Review by TheBusyBiscuit — APPROVED** [review]

File: pages/Magical-Gadgets.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+The Magical Gadgets contains various gadgets with magical powers. Use your magical items to create these!
```
Comment (LinoxGH) [inline pages/Magical-Gadgets.md:1]: Just a little suggestion: maybe use `craft` instead of `create`.

File: pages/Magical-Gadgets.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+[Reinforced Spawner](https://github.com/Slimefun/Slimefun4/wiki/Reinforced-Spawner)  
+[Scroll of Dimensional Teleposition](https://github.com/Slimefun/Slimefun4/wiki/Scroll-of-Dimensional-Teleposition)  
+[Tome of Knowledge Sharing](https://github.com/Slimefun/Slimefun4/wiki/Tome-of-Knowledge-Sharing)  
+[Flask of Knowledge](https://github.com/Slimefun/Slimefun4/wiki/Flask-of-Knowledge)  
+[Ancient Altar](https://github.com/Slimefun/Slimefun4/wiki/Ancient-Altar)  
+[Infernal Bonemeal](https://github.com/Slimefun/Slimefun4/wiki/Infernal-Bonemeal)  
+[Elytra Scale](https://github.com/Slimefun/Slimefun4/wiki/Elytra-Scale)  
+[Elytra](https://github.com/Slimefun/Slimefun4/wiki/Elytra)  
+[Infused Elytra](https://github.com/Slimefun/Slimefun4/wiki/Infused-Elytra)  
+[Soulbound Elytra](https://github.com/Slimefun/Slimefun4/wiki/Soulbound-Elytra)
```
Comment (LinoxGH) [inline pages/Magical-Gadgets.md:24]: I think these could be merged into an `Elytras` page.

**Review by LinoxGH — APPROVED** [review]
Just a couple little suggestion.

**Comment by svr333** [comment]
You can accept and merge so I can get a new PR in for hacktober :)

**Comment by TheBusyBiscuit** [comment]
You should be working on seperate branches anyway, which you're actually doing.
So you can do multiple PRs from independent branches.

**Review by LinoxGH — APPROVED** [review]

**Comment by svr333** [comment]
Well yes I know but I mentioned it specifically because I first said to leave this open since I was going to add more

File: pages/Magical-Gadgets.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+[Infused Elytra](https://github.com/Slimefun/Slimefun4/wiki/Elytras)  
+[Soulbound Elytra](https://github.com/Slimefun/Slimefun4/wiki/Elytras)  
+[Totem of Undying](https://github.com/Slimefun/Slimefun4/wiki/Totem-of-Undying)  
+[Rainbow Wool](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Blocks)  
+[Rainbow Glass](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Blocks)  
+[Rainbow Glass Pane](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Blocks)  
+[Rainbow Clay](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Blocks)  
+[Rainbow Concrete](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Blocks)  
+[Rainbow Glazed Terracotta](https://github.com/Slimefun/Slimefun4/wiki/Rainbow-Blocks)  
+[Infused Hopper](https://github.com/Slimefun/Slimefun4/wiki/Infused-Hopper)
```
Comment (TheBusyBiscuit) [inline pages/Magical-Gadgets.md:32]: I just realized... There are no bullet points...

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

**Review by LinoxGH — APPROVED** [review]

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**